specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/examples/WorkspaceVirtualNetworkPeeringCreate.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "workspaceName": "myWorkspace",
+    "peeringName": "vNetPeeringTest",
+    "api-version": "2018-04-01",
+    "VirtualNetworkPeeringParameters": {
+      "properties": {
+        "allowForwardedTraffic": null,
```

Comment:
should it be `true` or `false`?

Reply:
Yes, the example values I took it from our Portal request. When this value is null it the value gets resolved to default value which is "false". I have updated the example with false.